packages/services/api/src/modules/organization/module.graphql.ts

```diff
@@ -303,7 +307,6 @@ export default gql`
   extend type Member {
     canLeaveOrganization: Boolean!
     role: MemberRole!
-    isAdmin: Boolean!
```

Comment:
There is no single admin anymore. There is only an organization owner. All other members have permissions assigned based on their user role.